integration

```diff
@@ -405,6 +405,7 @@
   "h4de5/home-assistant-vimar",
   "ha-warmup/warmup",
   "HandyHat/ha-hildebrandglow-dcc",
+  "happydev-ca/evduty-home-assistant",
```

Comment:
PR is not editable by maintainers.

Reply:
Seems like it cant be done for forks from an org account.
https://github.com/orgs/community/discussions/5634